### PR TITLE
Automatic model selection

### PR DESCRIPTION
**Pull request by pavolloffay**
State is initialized as follows:
* double exponential smoothing - simple: (level0=y0, slope0=y1-y0), complex: fit of linear regression
* simple exponential smoothing - simple: (level0=y0), complex: mean(initData$y)

This implementations of exp. smoothing give lower MSE than R's ets().
Model selection and parameters estimation is much faster than OpenForecast

### CODE REVIEW

**Comment by pavolloffay** [comment]
Tests for continual learning needs to be added.

**Comment by pavolloffay** [comment]
@Jiri-Kremser  could you please review?

File: R/testData/trendStatDownwardHighVar.model [hunk 1]
```diff
@@ -1,3 +1,9 @@
...
-"l" "ETS(A,N,N)" 96448842.6794396 46.1801296372608
+model: ETS(A,N,N) 
+ MSE: 96448842.6794396 
+ aic: 5980.49605398248 
+ bic: 5987.5389758182 
+ aicc: 5980.54463697843 
+ alpha: 0.0553397015634794 
+ beta: NA 
+ l: 46.1801296372608 
+ b: NA
```
Comment (Jiri-Kremser) [inline R/testData/trendStatDownwardHighVar.model]: are those models generated by R and read by Java code?

Comment (pavolloffay) [inline R/testData/trendStatDownwardHighVar.model]: yes, then I compare them against my implementation

**Pull request merged**